de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/allocation/impl/InventoryAllocationDestination.java

```diff
@@ -232,6 +232,10 @@ private I_M_InventoryLine getCreateInventoryLine(final I_M_InOutLine inOutLine,
 
 		// set the M_HU_PI_Item_Product from the HU
 		inventoryLine.setM_HU_PI_Item_Product(hu.getM_HU_PI_Item_Product());
+		
+		final BigDecimal qtyTU = hu.getM_HU_LUTU_Configuration().getQtyTU();
```

Comment:
Nop, it can be null ...
e.g.
on w101, 
```
select count(1) from M_HU where M_HU_LUTU_Configuration_ID is null and M_HU_Item_Parent_ID is null" 
```
returned 502.

But btw @metas-ts  i think we shall have some API which is returning this information.
That API shall also cover the Aggregated HU case.

Atm, we have a similar API when we are building the HU display names... see de.metas.handlingunits.impl.HUDisplayNameBuilder.IncludedHUsCounter